src/EnergyPlus/StandardRatings.cc

```diff
@@ -3799,7 +3799,7 @@ namespace StandardRatings {
                 } else if (bl >= q_full) {
                     // CASE 3 : 11.2.2.3.3 AHRI-2023
                     // Building Load is greater than the capacity of the unit at the Full Compressor Speed, q_full <= bl or (bl >= q_full:)
-                    if (t > (-15) || t <= (-8.33)) {
+                    if (t > (-15) && t <= (-8.33)) {
```

Comment:
@Nigusse would you please look at this change to see if you agree? The CppCheck warning in question is:

    [src/EnergyPlus/StandardRatings.cc:3802]:(warning),[incorrectLogicOperator],Logical disjunction always evaluates to true: t > -15 || t <= -8.33.

Reply:
Yeah this way definitely _seems_ better now.   @Nigusse ?

Reply:
I looked at it and then did quick check of the standard `2023 AHRI 210/240`, the original logic syntax ` if (t > (-15) || t <= (-8.33)) {` is consistent with the standard. Needs further check because, I see something suspicious of the code under the block`} else if (bl >= q_full) { }`.

Reply:
See the standard snippet below:

![image](https://user-images.githubusercontent.com/5805126/225078623-5306fc96-ae00-408e-be3c-bc96e631e5da.png)

Further checking the standard the variable `𝑡_𝑂𝐵𝑂` is defined as `45F`, which is `7.22C`. With the `𝑡_𝑂𝐵𝑂` changed to `7.22` the original equation makes sense to me. Also the subsequent lines of code in this if block need further review.

Reply:
> I looked at it and then did quick check of the standard 2023 AHRI 210/240, the original logic syntax  if (t > (-15) || t <= (-8.33)) { is consistent with the standard.  
   \- @Nigusse 

You may be agreeing with the issue here, but I can't tell.  I'll just say I'm not sure how this logic could be consistent with anything.  The two conditions are 
 - t greater than -15
 - or
 - t <= -8.33

This is always true regardless of the value of t.  Negative infinity?  Yep, less than -8.  Negative ten?  Yep, less than -8.  Negative 5?  Yep, greater than -15.  Positive infinity?  Yep, greater than -15.  

Do we just need to eliminate this IF check entirely?  That's what the code is **doing** right now since it is always true anyway.

Reply:
I am still suspicious that all temperature bins are not covered when the building load is greater than the full capacity at the maximum compressor speed. I would rather post issue so that developers look it.

Reply:
I think this change should be reverted, an issue opened, and this section of code be revised to meet the standard intent.